files/nrpe/check_keystone

```diff
@@ -82,12 +90,12 @@ class CheckKeystone(keystone.Client):
     # list of unique email addresses
     uniq_users = list(set([user.email for user in users]))
     # filter CSC users
-    csc_users  = filter(lambda user: '@csc.fi' in user, uniq_users)
+    csc_users  = list(filter(lambda user: '@csc.fi' in user, uniq_users))
```

Comment:
Required to make this script work with python3, otherwise will get `TypeError: object of type 'filter' has no len()`